velox/functions/lib/window/RowNumber.cpp

```diff
@@ -39,8 +40,20 @@ class RowNumberFunction : public exec::WindowFunction {
       const SelectivityVector& validRows,
       vector_size_t resultOffset,
       const VectorPtr& result) override {
+    if (resultType_->isInteger()) {
+      applyBlock<int32_t>(peerGroupStarts, resultOffset, result);
+    } else {
+      applyBlock<int64_t>(peerGroupStarts, resultOffset, result);
+    }
+  }
+
+  template <typename T>
+  void applyBlock(
```

Comment:
naming: applyBlock name is a bit out of convention. Can this be just 'apply' or 'applyTyped'?

Reply:
Fixed.